infrastructure/lib/yjit_benchmarking.rb

```diff
@@ -113,7 +113,7 @@ def ensure_stopped!
       def execute
         ensure_stopped!
         with_instances(reporting_instance, state: allowed_states) do |instance|
-          ssh_exec(instance, "#{LAUNCH_SCRIPT} report")
+          ssh_exec(instance, "YJIT_METRICS_NAME=#{(opts[:ref] || "main").dump} #{LAUNCH_SCRIPT} report")
```

Comment:
Can we put this part into opts processing so we can have a default of `main` there?

Reply:
Indeed, good call.